deepchem/models/torch_models/infograph.py

```diff
@@ -101,81 +291,138 @@ class InfoGraphModel(ModularTorchModel):
     >>> ds = NumpyDataset(X, y, w)
     >>> num_feat = max([ds.X[i].num_node_features for i in range(len(ds))])
     >>> edge_dim = max([ds.X[i].num_edge_features for i in range(len(ds))])
-    >>> model = InfoGraphModel(num_feat, edge_dim, 15, use_unsup_loss=True, separate_encoder=True)
+    >>> model = InfoGraphStarModel(num_feat, edge_dim, 15, training_mode='semisupervised')
     >>> loss = model.fit(ds, nb_epoch=1)
```

Comment:
In the original infograph* paper, they provide the ability to learn from different sets of labeled and un-labeled examples. Do we have that capability in our implementation? We should add in that capability if we don't have it, since otherwise we can't use a larger unlabeled set to bootstrap training on a small labeled dataset

Reply:
Adding on to this previous comment, we should add a discussion in the docs about this limitation and point to the mismatch with the math in the original paper even if we don't have an easy fix for how to use two different datasets for labeled/unlabeled data

Reply:
Quick reminder to add this discussion